ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -2,5 +2,11 @@
 import ivy
 
 
-def cat(tensors, dim=0, *, out=None):
-    return ivy.concat(tensors, dim, out=out)
+def reshape(input, shape):
+    return ivy.expand_dims(input, shape)
```

Comment:
please use `ivy.reshape` instead of expand_dims

Reply:
updated to return ivy.reshape, as that is the main alias